uavcan/register/384.Access.1.0.uavcan

```diff
@@ -123,18 +162,49 @@
 # For example, a register named "uavcan.pub.measurement.type" may contain "uavcan.si.sample.angle.Quaternion.1.0".
 #
 # ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.diagnostic.*
+#
+# Prefix reserved for future use.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.can.bitrate                                  natural32[2]    implementation-defined  implementation-defined
+#   uavcan.can.iface                                    string          mutable, persistent     implementation-defined
+#
+# These registers are only relevant for nodes that support UAVCAN/CAN.
+#
+# uavcan.can.bitrate defines the CAN bus bit rate: the first value is the arbitration bit rate, the second is the
```

Comment:
do we not need sample points as well?

Reply:
IIRC, NXP were going to give this question a look in the context of DS-015 but it is yet to happen. For now, assume that the device is able to configure the timings appropriately automatically (this assumption is made by the [PythonCAN API](https://pyuavcan.readthedocs.io/en/latest/.apidoc_generated/pyuavcan.transport.can.media.pythoncan.html) in PyUAVCAN, for instance). Later we will probably need to add a dedicated register for sample point configuration.

@PetervdPerk-NXP

Reply:
I do concur with @thirtytwobits about having a sample point option as well. Or at least thinking about minimal sample point or something like that. MCU's can indeed calculate timings (e.g. https://github.com/apache/incubator-nuttx/blob/master/arch/arm/src/s32k1xx/s32k1xx_flexcan.c#L393). However only having the timing from just the bitrate isn't enough you might end up with a low sample point e.g. 60% which is non-ideal.

Regarding DS-015 we participated in the network topology discussion and stated if that if there's a well defined configuration of nodes and it's topology we could help analyzing the impact of different bitrates on such a topology.

Reply:
Moved this into a separate issue: https://github.com/UAVCAN/public_regulated_data_types/issues/110 any help from NXP would be appreciated.